packages/ui/A11yDecorator/A11yDecorator.js

```diff
@@ -60,66 +61,54 @@ const defaultConfig = {
 const A11yDecorator = hoc(defaultConfig, (config, Wrapped) => {
 	const {prop} = config;
 
-	return kind({
-		name: 'A11yDecorator',
+	// eslint-disable-next-line no-shadow
+	const A11yDecorator = React.forwardRef(({accessibilityHint, accessibilityPreHint, 'aria-label': ariaLabel, [prop]: content, ...rest}, ref) => {
```

Comment:
Let's use `function A11yDecorator () { ... }` to assign the name here rather than explicitly after the fact.

Also, can you expand on why we've included `forwardRef` here?